githubaction/0.1.0/README.md

```diff
@@ -0,0 +1,257 @@
+
+1. GitHub repo with a GitHub Action
+
+    The GitHub repository must have at least one action workflow. If you are not familiar with GitHub, we recommend you review the [GitHub getting started documentation](https://docs.github.com/en/actions/learn-github-actions).
+
+1. GitHub Action configured to support repository dispatch
+
+    Use the GitHub API to trigger a webhook event called [repository_dispatch](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#repository_dispatch) that will start a GitHub action workflow. 
+  
+    To use repository dispatches, configure your GitHub workflow with the `repository_dispatch` trigger and provide a dispatch name that will match to the value. Within the Keptn webhook payload that is sent to the GitHub action API request, the `event_type` attribute is how GitHub matches the request to the GitHub workflow to run.  
+    
+    Below is an example with a repository dispatch with the event type name of `silent`.
+
+    ```
+    name: webhook silent example
+    on:
+      repository_dispatch:
+        types: 
+          - silent
+    ```
+
+## Step 2: Store GitHub token as a Keptn secret
+
+To secure the GitHub Access Token, a Keptn secret must be created.  To do this, go to the Keptn project  `Uniform page > Secret` and click the `Add Secret` button.  On that form fill the following values:
+  * **Name**: Name such as `github-secret` 
+  * **Scope**: `keptn-webhook-service` 
+  * **Key-value pairs**:  Click the `new key-value pair` button and add these values:
+    * **Key** = Name such as `GITHUBTOKEN`
+    * **Value** = The GitHub Access Token from the step from previous section 
+
+The completed form should look as follows:
+
+<img src="images/createsecret.png" width="50%" height="50%">
+
+## Step 3: Create Keptn project
+
+You can incorporate GitHub webhooks into any Keptn project, but the example use cases in the next section assume you have created a Keptn project called `demo` with a [shipyard file](https://keptn.sh/docs/0.10.x/manage/shipyard/) as follows and a Keptn service called `casdemoapp`:
+
+```
+apiVersion: spec.keptn.sh/0.2.2
+kind: "Shipyard"
+metadata:
+  name: "demo-webhook"
+spec:
+  stages:
+    - name: "production"
+      sequences:
+      - name: "mysequence"
+        tasks: 
+        - name: "mytask-interactive"
+```
+
+# GitHub integration Webhook options
+
+There are two use cases for Keptn webhooks:
+
+1. **Silent** - The webhook subscription listens for status events, `started` or `finished`, and calls the GitHub Action API to invoke the GitHub workflow. GitHub does not need to send anything back to Keptn and the Keptn sequence will just continue to process.
+1. **Interactive** -  The webhook subscription listens for`triggered` events, calls the GitHub Action API to invoke the GitHub workflow. Once it does the  Keptn sequence will wait until the GitHub workflow to send back a `finished` event.
```

Comment:
space between: for`triggered`

Reply:
fixed in https://github.com/keptn-sandbox/artifacthub/pull/34/commits/a0ab4afe44744d3ee98494d4f368c60a5f18bea4